src/components/Input.tsx

```diff
@@ -0,0 +1,24 @@
+import { useState} from 'react';
+import Button from './Button';
+
+
+const Input=()=> {
+    const [value, setValue] = useState<string>("");
+
+    function handleChange(event: React.ChangeEvent<HTMLInputElement>) {
```

Comment:
Change this to an arrow function too!

Reply:
Made the change. Will keep in mind to use arrow functions for callbacks !